### PR TITLE
Fix #214 Instructions for highway in cocos2dx

### PR DESCRIPTION
**Pull request by EugeneButusov**

### CODE REVIEW

**Comment by EugeneButusov** [comment]
@borisspektor, is here code examples or some another integration steps should be changed or is it enough?

**Comment by borisspektor** [comment]
@EugeneButusov this is enough for now. Please add keeva to core docs as well, and in HW docs add a note that if core is present and keeva is present in core no need to add it in HW as well.

**Comment by EugeneButusov** [comment]
@borisspektor, did that.

File: src/documents/cocos2dx/cpp/grow/GrowSpend_GettingStarted.html.md [hunk 1]
```diff
@@ -213,6 +219,8 @@ That's it! Now all you have to do is build your XCode project and run your game.
     - SoomlaAndroidCore.jar
 
     - Cocos2dxAndroidCore.jar
+    
+    - keeva-1.0.1.jar (not required if already added in `cocos2dx-highway`)
```
Comment (borisspektor) [inline src/documents/cocos2dx/cpp/grow/GrowSpend_GettingStarted.html.md]: @EugeneButusov just leave keeva, no need for the `not required` part

Comment (EugeneButusov) [inline src/documents/cocos2dx/cpp/grow/GrowSpend_GettingStarted.html.md]: @borisspektor, done.

**Pull request merged**